packages/core/core/telemetry/get-tracer.ts

```diff
@@ -0,0 +1,23 @@
+import opentelemetry, { Tracer } from '@opentelemetry/api';
```

Comment:
Do `import { traceApi, Tracer } from '@opentelemetry/api'` to use `traceApi.getTracer(...)`. I believe this has a better chance of tree shake more code out of the binary.